bots/zephyr_mirror_backend.py

```diff
@@ -596,13 +623,14 @@ def zcrypt_encrypt_content(zephyr_class, instance, content):
     return encrypted
 
 def forward_to_zephyr(message):
+    # type: (Dict) -> zulip
```

Comment:
There is no `zulip` class anywhere in the code base. It is a `module` but it couldn't be a `type`.